### DISCUSSION TITLE
Manual Discussions

### DISCUSSION THREAD

**Discussion by coverbeek** [frc5567/2023-Robot#133] [Team Posts]
Use this thread to note details in the game manual that may be of interest.

Specifically, things like the fact that we are going to have dedicated video streaming at least for "sandstorm" period of match based on the details on page 32. However, bandwidth will potentially still be an issue, per note on page 84 that we're restricted to a total of 4mbit/s

**Comment by MikasaSucasa** [comment]
How does this bandwidth compare to past years? Have we ever streamed two cameras before? *Cough* Returners

**Comment by RugBark** [comment]
This year we only have about half the available bandwidth from last year (7Mbps -> 4Mbps). We streamed two cameras in Steamworks.
